meta/contao/installation-bundle/de.yml

```diff
@@ -0,0 +1,5 @@
+de:
+    title: Contao Installtool
```

Comment:
Depending on wether we see "Contao Install Tool" as a name such as "Contao Managed Edition", this should be written as "Contao Install Tool". 🤷‍♂

Reply:
well that's what I had in the first place 😂

Reply:
I am fine with either "Contao Install Tool" or "Contao Installtool". As long as it is not a third variant. 😄

Reply:
But then it should be changed in the `contao/installation-bundle` as well, because there "Contao-Installtool" is used 😬.